package.json

```diff
@@ -7,17 +7,20 @@
     "clean": "rm -rf dist/ && rm -rf generated/",
     "start": "webpack-dev-server --config utils/build/webpack.dev.js",
     "build": "webpack --config utils/build/webpack.prod.js",
+    "server": "",
```

Comment:
I think I would remove this if it does nothing

Reply:
It does nothing but is referenced in the cypress run script so that when a server command is added, it should all come together

Reply:
Is this intended to be the mock admin server? If so, can you update it to `start:mockserver` (as this is what I will be updating it to be) and echo https://github.com/strimzi/strimzi-ui/issues/36 ?

Reply:
This is supposed to be the express server that serves the UI. With the current state of this PR, you have to stand up the dev server separately and then run cypress.

Reply:
Were you thinking it would serve the built UI, or dev (webpack dev server) UI? In either case, both would also need mock admin, and I am happy to put that together in #36

Reply:
I was thinking the built UI (to prevent accidental changes from being built while the tests are running). The cypress command has the scripts set up to clean build the UI, it just needs to be served. So yes, it would also need the mock server to be stood up too